modal/image.py

```diff
@@ -120,6 +129,39 @@ def _make_pip_install_args(
     return args
 
 
+async def _get_builder_version() -> ImageBuilderVersion:
+    async def lookup_version() -> str:
+        client = await _Client.from_env()
+        resp = await client.stub.ImageBuilderVersionLookup(Empty())
+        return resp.version
```

Comment:
The `Resolver` object in load() has a `Resolver.client` property, if you want to reuse that.